examples/TestInclusions/Podfile

```diff
@@ -0,0 +1,12 @@
+workspace 'Examples.xcworkspace'
+project 'TestInclusions.xcodeproj'
+platform :ios, '6.0'
+
+abstract_target 'TestInclusionsPods' do
+    pod 'SwrveConversationSDK', '~> 4.3.0'
+
+    target 'TestInclusions'
+    target 'TestInclusionsTests' do
+        pod 'OCHamcrest', '~> 5.0'
```

Comment:
For the sake of simplicity, I'd avoid unnecessary dependencies here and would rather use XCTest's built-in matchers.